explorer/nodewatch/nodewatch/NetworkRepository.py

```diff
@@ -192,6 +198,15 @@ def _create_descriptor_from_json(self, json_node):
 		if self._network.generation_hash_seed != Hash256(json_node['networkGenerationHashSeed']):
 			return None
 
+		api_node_info_data = (None, None, None, None)
+		if 'apiNodeInfo' in json_node:
```

Comment:
is there a reason you chose to add 'apiNodeInfo' instead of including this in 'extraData'